src/resources/profiles.jsx

```diff
@@ -0,0 +1,83 @@
+import { 
+    List, 
+    TextField, 
+    EmailField,
+    DateField,
+    Datagrid,
+    Edit,
+    SimpleForm,
+    TextInput,
+    DateTimeInput,
+    RadioButtonGroupInput,
+    SelectField,
+    FunctionField,
+    Create,
+    Show,
+    SimpleShowLayout
+} from "react-admin";
+
+const profileFilters = [
+  <TextInput source="q" label="Search" alwaysOn />,
+];
+
+const gender = [
+  { id: 0, name: "male" },
+  { id: 1, name: "female" },
+  { id: 2, name: "divers" },
+  { id: null, name: "no gender" },
```

Comment:
"no gender"? lieber "not specified" haha

Reply:
thats what it's called in the app just copied that

Reply:
In the app, the choice is "Prefer Not To Say". We only show <No Gender> in the show page there to be consistent with all the other fields. But here in ReactAdmin those are shown differently anyways